nuxt.config.ts

```diff
@@ -225,8 +222,28 @@ export default defineNuxtConfig({
     '@pinia/nuxt',
     '@nuxt/content',
     'nuxt-simple-sitemap',
+    '@nuxtjs/google-fonts',
+    'nuxt-purgecss',
```

Comment:
still broken on deployment preview, I suspect because of this module. let's try it by removing the `nuxt-purgecss` module

Reply:
ye, it was because of this, it has `enable: false` for dev env, so I didn't notice problem